src/main/kotlin/no/nav/tilleggsstonader/sak/arbeidsfordeling/ArbeidsfordelingService.kt

```diff
@@ -9,38 +16,64 @@ import org.springframework.stereotype.Component
 class ArbeidsfordelingService(
     @Qualifier("shortCache")
     private val cacheManager: CacheManager,
+    private val arbeidsfordelingClient: ArbeidsfordelingClient,
+    private val personService: PersonService,
+    private val egenAnsattService: EgenAnsattService,
 ) {
 
     companion object {
         const val MASKINELL_JOURNALFOERENDE_ENHET = "9999"
         val ENHET_NASJONAL_NAY = Arbeidsfordelingsenhet("4462", "Tilleggsstønad INN")
     }
 
-    fun hentNavEnhetId(ident: String, oppgavetype: Oppgavetype) = when (oppgavetype) {
-        Oppgavetype.VurderHenvendelse -> hentNavEnhetForOppfølging(ident, oppgavetype)?.enhetId
-        else -> hentNavEnhet(ident)?.enhetId
+    fun hentNavEnhetId(ident: String, oppgavetype: Oppgavetype, tema: Tema = Tema.TSO) = when (oppgavetype) {
```

Comment:
Ønsker vi at TSO skal være default generelt? Er greit for nå fordi vi ikke har noe annet, men burde kanskje se på dette senere når det blir flere

Reply:
TSO kan vere default, men det er gjelder-verdien som me må sjekke at blir ruta korrekt ut frå korleis me legger opp saksbehandlinga (når me begynner på daglig reise så kjem denne problematikken)